addons/sale_management/views/sale_portal_templates.xml

```diff
@@ -13,16 +13,9 @@
             <t t-set="display_remove" t-value="any([line.sale_order_option_ids for line in sale_order.order_line])"/>
         </xpath>
 
-        <xpath expr="//section[@id='details']//table[@id='sales_order_table']/thead/tr" position="inside">
```

Comment:
I think this part needs to be merged in the basic template, no completely removed (cfr comment on task)

Reply:
@dbo Done